### PR TITLE
Add Shirley Temples, Start of modularization

### PR DESCRIPTION
**Pull request by MitztheKat**
## About The Pull Request

Adds a new folder for modularizing Tegu code.
Adds Shirley Temples.
I know how to code now ^-^

## Why It's Good For The Game

PRAISE BE SHIRLEY TEMPLES!

### CODE REVIEW

File: modular_tegu/modules/tegu_food_drink/tegufood.dm [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/datum/reagent/drink/tegu/shirley
```
Comment (Cupax3) [inline modular_tegu/modules/tegu_food_drink/tegufood.dm:1]: As Egor mentioned, no need for the /tegu/, you can just make it a normal drink.

File: baystation12.dme [hunk 2]
```diff
@@ -3213,5 +3213,6 @@
 #include "maps\~mapsystem\maps_jobs.dm"
 #include "maps\~mapsystem\maps_unit_testing.dm"
 #include "maps\~unit_tests\unit_testing.dm"
+#include "modular_tegu\modules\tegu_food_drink\tegufood.dm"
```
Comment (Cupax3) [inline baystation12.dme:3216]: No need for modularization, please add it to the main foods/drinks .dm

**Review by Cupax3 — CHANGES_REQUESTED** [review]
See comments, no modules needed, feel free to add it directly to the main drinks .dm

**Comment by Cupax3** [comment]
Also-Also, please add :cl: changelog, thankies

**Comment by MitztheKat** [comment]
I'll just close this and re-open a better one.

**Pull request closed without merging**